docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -11,42 +28,32 @@ the data locally in [MOLGENIS Armadillo](/#/armadillo/) DataSHIELD server.
 
 ### Staging areas for uploads
 
-The metadata of the cohort or of the data source is uploaded to what are called "staging areas" of the Data Catalogue.
+The metadata of a cohort or data source is uploaded to what are called "staging areas" of the Data Catalogue.
 You will need credentials to log in and upload metadata.  
 LifeCycle, ATHLETE and LongITools use [*data-catalogue-staging*](https://data-catalogue-staging.molgeniscloud.org).  
-ConcePTION uses [*emx2-conception*](https://emx2-conception.test.molgenis.org).
+ConcePTION uses [*conception-acc*](https://conception-acc.molgeniscloud.org).
 
 When you log in, you will be able to at least see the following databases:
 
-- Catalogue: The common data models of your consortia, which you need to use in the mappings or ETL.
+- DataCatalogue: The catalogue data, in which you can search for target variables to map to.
 - CatalogueOntologies: This database contains the look-up list that you need for filling out some columns in the
   templates, e.g. format or unit. If you need to add anything to these look-up lists, contact us
   at [molgenis-support](mailto:molgenis-support@umcg.nl).
+- SharedStaging: A communal staging area in which Contacts and Institutions are added and edited.
 - The database of your own cohort or data source, that you can use to upload the templates once you have filled them
   out.
 
 ![MOLGENIS databases](../img/cat_databases.png)
 
 <sup>*Figure 1. Databases in the Data Catalogue staging area.*</sup>
 
-### Define metedata of cohorts or data sources
+### Fill out cohort rich metadata
 
-[MOLGENIS Data Catalogue](https://data-catalogue.molgeniscloud.org/catalogue/catalogue/#/variable-explorer/) provides a
-framework to describe in detail metadata of cohorts and of data sources, definitions of the variables collected in
-cohorts and in the data banks composing the data sources (aka 'source variables'), and mappings to common data models (
-aka 'target variables'). Its purpose is to facilitate pooled data analysis of multiple cohorts together, and multi-data
-source studies [Gini et al, 2020](https://pubmed.ncbi.nlm.nih.gov/32243569/).
+Open your staging area, navigate to 'Tables' and open the table 'Cohorts'. Your cohort pid and name are already 
```

Comment:
A cannot find the table 'Cohorts'  when I open testCohort on data-catalogue-staging

Reply:
The set up is not implemented on data-catalogue-staging yet. Please check brenda2.molgeniscloud.org, e.g. schema testStaging2